gcloud-java-resourcemanager/README.md

```diff
@@ -27,27 +27,145 @@ Example Application
+
+```java
+import com.google.gcloud.resourcemanager.ResourceManager;
+import com.google.gcloud.resourcemanager.ResourceManagerOptions;
+
+ResourceManager resourceManager = ResourceManagerOptions.defaultInstance().service();
+```
+
+#### Creating a project
+All you need to create a project is a globally unique project ID.  You can also optionally attach a non-unique name and labels to your project. Read more about naming guidelines for project IDs, names, and labels [here](https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects). To create a project, add the following import at the top of your file:
+
+```java
+import com.google.gcloud.resourcemanager.ProjectInfo;
+```
+
+Then add the following code to create a project (be sure to change `myProjectId` to be something unique).
+
+```java
+String myProjectId = "my-globally-unique-project-id"; // Change to a unique project ID.
+ProjectInfo myProject = resourceManager.create(ProjectInfo.builder(myProjectId).build());
+```
+
+Note that the return value from `create` is a `ProjectInfo` that includes additional read-only information, like creation time, project number, and lifecycle state. Read more about these fields on the [Projects page](https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects).
+
+#### Getting a specific project
+You can load a project if you know it's project ID and have read permissions to the project. For example, to get the project we just created we can do the following:
+
+```java
+ProjectInfo projectFromServer = resourceManager.get(myProjectId);
+```
+
+#### Editing a project
+To edit a project, create a new `ProjectInfo` object and pass it in to the `ResourceManager.replace` method.
+
+For example, to add a label for the newly created project to denote that it's launch status is "in development", add the following code:
+
+```java
+ProjectInfo newProjectInfo = resourceManager.replace(projectFromServer.toBuilder()
+    .addLabel("launch-status", "in-development").build());
+```
+
+Note that the values of the project you pass in to `replace` overwrite the server's values for non-read-only fields, namely `projectName` and `labels`. For example, if you create a project with `projectName` "some-project-name" and subsequently call replace using a `ProjectInfo` object that didn't set the `projectName`, then the server will unset the project's name. The server ignores any attempted changes to the read-only fields `projectNumber`, `lifecycleState`, and `createTime`. The `projectId` cannot change.
+
+#### Listing all projects
+Suppose that we want a list of all projects for which we have read permissions. Add the following import:
```

Comment:
Yes, I can view read-only projects using "get" and "list" even though the oauth scope is read/write